nsv13/code/modules/cargo/packs.dm

```diff
@@ -640,6 +640,15 @@
 					/obj/item/extinguisher/advanced/hull_repair_juice)
 	crate_name = "engineering crate"
 
+/datum/supply_pack/security/taser
+	name = "Taser Crate"
+	desc = "From the depths of stunbased combat, this order rises above, supreme. Contains three hybrid tasers, capable of firing both electrodes and disabling shots. Requires Security access to open."
+	cost = 3000
+	contains = list(/obj/item/gun/energy/e_gun/advtaser,
+					/obj/item/gun/energy/e_gun/advtaser,
+					/obj/item/gun/energy/e_gun/advtaser)
+	crate_name = "taser crate"
+
```

Comment:
```suggestion

```
These can stay out, they also double as lasers and we already have our own tasers anyway

Reply:
The big difference between tazer types aside from damage is that ours are hitscan. The old ones had relatively slow projectiles that you could kinda dodge.

Reply:
Ours are short-range to compensate for that though, but it's an interesting dichotomy nonetheless